src/components/ErrorMessage/Footer/Footer.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { TodoListState } from '../../../types/TodoListState';
+
+interface FooterProps {
+  activeTodos: number;
+  completedTodos: number;
+  filter: string;
+  setFilter: (newFilter: TodoListState) => void;
+  clearCompleted: () => void;
+}
+
+export const Footer: React.FC<FooterProps> = ({
+  activeTodos,
+  completedTodos,
+  filter,
+  setFilter,
+  clearCompleted,
+}) => {
+  const handleFilterClick = (
+    event: React.MouseEvent<HTMLAnchorElement>,
+    filterType: TodoListState,
+  ) => {
+    event.preventDefault();
+    setFilter(filterType);
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${activeTodos} items left`}
```

Comment:
Consider using a more descriptive message when there is only one item left for better user experience.